src/cli.js

```diff
@@ -46,7 +46,10 @@ function CLI(argv) {
 
       console.info(combinedSchema.toString());
     })
-    .catch(console.error);
+    .catch(error => {
+      console.error(error)
```

Comment:
For readability reasons I would suggest to output only the `error.message` here.